### PR TITLE
Prettier config

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "babel",
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "trailingComma": "none",
```
Comment (krulis-martin) [inline .prettierrc]: You do not like trailing commas? I do. :)

Comment (Neloop) [reply .prettierrc]: I have no strong opinion about them. Currently our setup of prettier was this, but we can surely sure change it...

Comment (krulis-martin) [reply .prettierrc]: I would prefer `es5` value. I think it creates much less error prone environment (as we do not constantly forget writing commas when appending items to objects).

Comment (Neloop) [reply .prettierrc]: Ok

**Pull request merged**